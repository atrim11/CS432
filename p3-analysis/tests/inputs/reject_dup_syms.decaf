def int main() {
    return func(-1, true, false, 10, 12, 23, 7);
}

def int func(int a, int a, int a, int a, int b, int c, void d) {
    bool a;
    int a;
    void a;
    bool b;
    bool b;
    int b;
    void c;
    void c;
    int c;
    bool d;
    bool c;
    return 1;
}