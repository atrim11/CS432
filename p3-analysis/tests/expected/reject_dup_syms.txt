Type mismatch in parameter 1 of call to 'func': expected int but found bool on line 2
Type mismatch in parameter 2 of call to 'func': expected int but found bool on line 2
Type mismatch in parameter 6 of call to 'func': expected void but found int on line 2
Void variable 'a' on line 8
Void variable 'c' on line 12
Void variable 'c' on line 13
Duplicate symbols named 'a' in scope started on line 5
Duplicate symbols named 'b' in scope started on line 5
Duplicate symbols named 'c' in scope started on line 5
Duplicate symbols named 'a' in scope started on line 5
